### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by dstathis** [nwg-piotr/nwg-displays#64]
I get an error when trying to start nwg-displays. This doesn't seem to always happen but it happens a lot. I can't figure out how to reproduce it.

I get the following output:
```
$ nwg-displays 
Settings: {'view-scale': 0.15, 'snap-threshold': 10, 'indicator-timeout': 500, 'custom-mode': [], 'use-desc': False}
Running on Hyprland
Traceback (most recent call last):
  File "/nix/store/i0asn8zhcjjm9n91jyjff8k9h46hg85m-nwg-displays-0.3.19/bin/.nwg-displays-wrapped", line 9, in <module>
    sys.exit(main())
  File "/nix/store/i0asn8zhcjjm9n91jyjff8k9h46hg85m-nwg-displays-0.3.19/lib/python3.10/site-packages/nwg_displays/main.py", line 1252, in main
    create_display_buttons()
  File "/nix/store/i0asn8zhcjjm9n91jyjff8k9h46hg85m-nwg-displays-0.3.19/lib/python3.10/site-packages/nwg_displays/main.py", line 569, in create_display_buttons
    outputs = list_outputs()
  File "/nix/store/i0asn8zhcjjm9n91jyjff8k9h46hg85m-nwg-displays-0.3.19/lib/python3.10/site-packages/nwg_displays/tools.py", line 183, in list_outputs
    outputs_dict[key]["monitor"] = monitors[idx]
IndexError: list index out of range
```

**Comment by arminius-smh** [comment]
Seems like the same crash that was fixed in 0.3.20 if you look at the linked issue: https://github.com/nwg-piotr/nwg-displays/releases/tag/v0.3.20

**Comment by dstathis** [comment]
Yup looks the same. I am currently running 0.3.19. Thanks.
